Logstash/logstash_http.conf

```diff
@@ -19,52 +24,100 @@ filter {
 
 	#ruby code to convert the comma seperated roles into an array
 	ruby{
-	     code =>"
+	    code =>"
 
-		 # method to split the supplied string by comma, trim whitespace and return an array
-		 def mapStringToArray(strFieldValue)
+		# method to split the supplied string by comma, trim whitespace and return an array
+		def mapStringToArray(strFieldValue)
 
-		    #if string is not null, return array
-		    if (strFieldValue != nil)
-		        fieldArr =  strFieldValue.split(',').map(&:strip).reject(&:empty?).uniq 
-		        return fieldArr                             
-		    end     
+		#if string is not null, return array
```

Comment:
issues in comment line. look for other places as well

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14